packages/x-components/src/utils/types.ts

```diff
@@ -163,6 +163,17 @@ export type EventsForDirectionLimit = {
  */
 export type ListItem = Identifiable & NamedModel;
 
+/**
+ * The type returned by the {@link gridItems} function. Basically it's a list of items with its CSS classes and a slotName
```

Comment:
I would just leave this grid item as a private type for the base grid component. It should not be part from the public API.

Reply:
Moreover the link does not work here, and causes the build to fail.